transformers4rec/torch/model/base.py

```diff
@@ -481,6 +482,7 @@ def __init__(
         head_reduction: str = "mean",
         optimizer: Type[torch.optim.Optimizer] = torch.optim.Adam,
         name: str = None,
+        max_sequence_length: Optional[int] = None,
```

Comment:
Added a `max_sequence_length` to limit the size of the padding when receiving ragged inputs.